### PR TITLE
Similarity updates

### PR DESCRIPTION
**Pull request by EthanMarx**
Updates to similarity data and model to handle new `asd` context.

### CODE REVIEW

File: amplfi/train/losses.py [hunk 1]
```diff
@@ -1,46 +1,129 @@
...
+        x:
+            Tensor with shape (batch_size, ..., dim).
+        eps:
+            Epsilon for numerical stability.
+
+    Returns:
+        The computed VICReg variance loss.
+    """
+    std = torch.sqrt(x.var(dim=0) + eps)
+    loss = torch.mean(F.relu(1.0 - std))
```
Comment (deepchatterjeeligo) [inline amplfi/train/losses.py:102]: make this `1.0` and keyword arg called `max_std`.

**Review by deepchatterjeeligo — CHANGES_REQUESTED** [review]
Looks good overall. Can you add a unittest for the VICReg loss. The test can take as input samples from gaussian distribution centered at say 10 and -10 each with unit variances.

**Comment by EthanMarx** [comment]
@deepchatterjeeligo both todos are done

**Review by deepchatterjeeligo — APPROVED** [review]
Thank you @EthanMarx
